src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -34,20 +34,20 @@
               kbn-typeahead-input
               ng-model="model.query"
               placeholder="Filter..."
-              aria-label="Filter input"
+              aria-label="Enter query"
```

Comment:
Makes more sense to me.  Do @Bargs or @lukasolson want to weigh in since this terminology is more in your area?